### PR TITLE
Org-level units preference

### PR DESCRIPTION
**Pull request by rgm**
#### Any background context you want to provide?

This is in preparation to adding metric units and is a (hopefully) easy PR to use to shake out our review and merge process, code quality, test requirements etc.

#### What's this PR do?

Adds an enum setting to Organizations in the database, and adds a select in the organization settings to specify the preferred display and import units for users under that organization.

#### How should this be manually tested?

Apply migrations. Under `/app/#/accounts/n` for some org id, confirm the select is set to "United States customary units". Uncomment the metric setting (see 3a2a1af6) and try to save the preferred measurement as "Metric", coming back to the screen later to confirm it saves and reloads.

#### What are the relevant tickets?

n/a.

#### Screenshots (if appropriate)

screen shot 2017-08-21 at 11 58 32

#### Definition of Done:
- [ ] Is there appropriate test coverage? (e.g. ChefSpec, Mocha/Chai, Python, etc.)
- [ ] Does this PR require a Selenium test? (e.g. Browser-specific bugs or complicated UI bugs)
- [x] Does this PR require a regression test? All fixes require a regression test.
- [x] Does this add new dependencies? If so, does PIP, npm, bower requirements need to be updated?

### CODE REVIEW

File: seed/lib/superperms/orgs/migrations/0004_organization_measurement_system.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ('orgs', '0003_auto_20160412_1123'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organization',
+            name='measurement_system',
+            field=models.IntegerField(choices=[(1, b'United States customary units'), (2, b'Metric (SI) units')], default=1),
```
Comment (nllong) [inline seed/lib/superperms/orgs/migrations/0004_organization_measurement_system.py]: I think we want the USC to be termed IP Units.

File: seed/lib/superperms/orgs/migrations/0005_auto_20170819_0952.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    operations = [
+        migrations.RemoveField(
+            model_name='organization',
+            name='measurement_system',
+        ),
+        migrations.AddField(
+            model_name='organization',
+            name='measurement_system_display',
+            field=models.IntegerField(choices=[(1, b'United States customary units - kBtu/sq ft'), (2, b'Metric (SI) units - GJ/m2'), (3, b'Metric (SI) units - kWh/m2')], default=1),
```
Comment (nllong) [inline seed/lib/superperms/orgs/migrations/0005_auto_20170819_0952.py]: kBtu/sq ft/year

File: seed/tests/test_account_views.py [hunk 3]
```diff
@@ -65,6 +66,8 @@ def test_dict_org(self):
                 'id': self.user.pk}],
             'number_of_users': 1,
             'name': 'my org',
+            'measurement_system_import': MEASUREMENT_US,
```
Comment (nllong) [inline seed/tests/test_account_views.py]: MEASUREMENT_IP

File: seed/lib/superperms/orgs/migrations/0004_organization_measurement_system.py (hunk 1 above)
Comment (rgm) [reply seed/lib/superperms/orgs/migrations/0004_organization_measurement_system.py]: Dealt with in a new migration.

Generally, is migration history OK or do you prefer it if I just alter the existing migration?

File: seed/lib/superperms/orgs/migrations/0005_auto_20170819_0952.py (hunk 2 above)
Comment (rgm) [reply seed/lib/superperms/orgs/migrations/0005_auto_20170819_0952.py]: (new migration, same comment as above).

**Comment by rgm** [comment]
No need to do this separately; I'll pull this into the revised PR based on our Sep 6 meeting.

**Pull request closed without merging**